container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -2215,7 +2755,7 @@ func (manager *KubernetesManager) waitForPodDeletion(ctx context.Context, namesp
 	)
 }
 
-func (manager *KubernetesManager) waitForPodTermination(ctx context.Context, namespaceName string, podName string) error {
+func (manager *KubernetesManager) WaitForPodTermination(ctx context.Context, namespaceName string, podName string) error {
```

Comment:
So this file is 2.7k lines, which is pretty huge. Might be worth contemplating (in a separate PR hehe) splitting these functions out into multiple files (which Go allows you to do).